source/libraries/ContractDeployer.ts

```diff
@@ -187,6 +188,18 @@ export class ContractDeployer {
         await Promise.all(promises);
     }
 
+    private async createGenesisUniverse(): Promise<ContractBlockchainData> {
```

Comment:
`approveCentralAuthority` and `getStakeToken` were actually moved down to below the TODO/CONSIDER comments, this wasn't moved up but Git sees this moving up as the smaller change.